amplify/auth/resource.ts

```diff
@@ -0,0 +1,7 @@
+import { defineAuth } from '@aws-amplify/backend';
+
+export const auth = defineAuth({
```

Comment:
Why do we need this?

Reply:
`auth` is a required field for defining the Amplify backend - it's not required by the typescript definitions but the deployment fails if it isn't there